winui/Kanban/column.md

```diff
@@ -0,0 +1,172 @@
+        <kanban:KanbanColumn HeaderText="Done" Categories="Done, Validated" />
+</kanban:SfKanban>
+
+{% endhighlight %}
+
+{% highlight C# hl_lines="4" %}
+
+this.kanban.Columns.Add(new KanbanColumn() { HeaderText = "To Do", Categories = "Open" });
+this.kanban.Columns.Add(new KanbanColumn() { HeaderText = "In Progress", Categories = "In Progress" });
+this.kanban.Columns.Add(new KanbanColumn() { HeaderText = "Code Review", Categories = "Code Review" });
+this.kanban.Columns.Add(new KanbanColumn() { HeaderText = "Done", Categories = "Done, Validated" });
+
+{% endhighlight %}
+{% endtabs %}
+
+## Customizing column headers
+
+### Customizing column header text
+
+You can set the header text for a column using the `HeaderText` property, which allows you to display simple text as the column title.
+
+{% tabs %}
+{% highlight XAML %}
+
+<kanban:KanbanColumn Categories="Open" HeaderText="To Do"/>
+
+{% endhighlight %}
+
+{% highlight C# %}
+
+this.kanban.Columns.Add(new KanbanColumn() { Categories = "Open", HeaderText = "To Do" });
+
+{% endhighlight %}
+{% endtabs %}
+
+### Customizing column header template
+
+To replace the UI of the header, you can use the `ColumnHeaderTemplate` of the `SfKanban`. The following code example illustrates how this can be done.
+
+{% tabs %}
+{% highlight XAML %}
+
+<kanban:SfKanban x:Name="kanban"
+                 ItemsSource="{Binding TaskDetails}">
+    <kanban:SfKanban.ColumnHeaderTemplate>
+        <DataTemplate>
+            <StackPanel Background="LightBlue">
+                <TextBlock Margin="10" Text="{Binding  HeaderText}" Foreground="Red" HorizontalAlignment="Center"/>
+            </StackPanel>
+        </DataTemplate>
+    </kanban:SfKanban.ColumnHeaderTemplate>
+    <kanban:SfKanban.DataContext>
+        <local:ViewModel/>
+    </kanban:SfKanban.DataContext>
+</kanban:SfKanban>
+
+{% endhighlight %}
+{% endtabs %}
+
+## Expanding or collapsing column
+
+To expand or collapse a column programmatically, use the `KanbanColumn.IsExpanded` property. Set this property to `true` to expand the column or `false` to collapse it.
+
+{% tabs %}
+{% highlight XAML %}
+
+<kanban:KanbanColumn Categories="Open" HeaderText="To Do" IsExpanded="False"/>
+
+{% endhighlight %}
+
+{% highlight C# %}
+
+this.kanban.Columns.Add(new KanbanColumn() { Categories = "Open", HeaderText = "To Do", IsExpanded = false });
+
+{% endhighlight %}
+{% endtabs %}
+
+## Enabling or disabling drag-and-drop
+
+The WinUI Kanban Control allows you to enable or disable the drag-and-drop functionality for cards using the `KanbanColumn.AllowDrag` property.
+
+{% tabs %}
+{% highlight XAML %}
+
+<kanban:KanbanColumn Categories="Open" HeaderText="To Do" AllowDrag="False" AllowDrop="False"/>
+
+{% endhighlight %}
+
+{% highlight C# %}
+
+this.kanban.Columns.Add(new KanbanColumn() { Categories = "Open", HeaderText = "To Do", AllowDrag = false, AllowDrop = false });
+
+{% endhighlight %}
```

Comment:
Work-in-Progress (WIP) limit and indications

The WinUI Kanban board allows users to set limits on the number of tasks in each column at every stage of the workflow, helping to prevent overload. The minimum and maximum number of items allowed in a column can be customized using the MinimumLimit and MaximumLimit properties in KanbanColumn.

The Kanban board also supports built-in error bars that change color when the WIP limit is exceeded or when a task fails validation, making it easier to quickly identify bottlenecks in the process.

Reply:
Content and example codes added